centreon/tests/e2e/cypress/e2e/Authentication/Custom-uri/01-custom-uri-authentication/index.ts

```diff
@@ -0,0 +1,110 @@
+import { Then, When } from '@badeball/cypress-cucumber-preprocessor';
+
+const service = 'Ping';
+const host = 'Centreon-Server';
+
+before(() => {
+  cy.startWebContainer();
+});
+
+beforeEach(() => {
+  cy.intercept({
+    method: 'GET',
+    url: '/monitor/api/internal.php?object=centreon_topology&action=navigationList'
+  }).as('getNavigationList');
+  cy.intercept({
+    method: 'GET',
+    url: '/monitor/include/common/userTimezone.php'
+  }).as('getTimeZone');
+  cy.intercept({
+    method: 'GET',
+    url: '/monitor/api/latest/monitoring/resources/hosts/*/services/*'
+  }).as('getResourceDetails');
+});
+
+When(
+  'I update the base uri within the corresponding web server configuration file',
+  () => {
+    cy.execInContainer({
+      command:
+        'bash -c "sed -i \'0,/centreon/s//monitor/\' /etc/httpd/conf.d/10-centreon.conf"',
+      name: Cypress.env('dockerName')
+    });
+  }
+);
+
+When('I reload the web server', () => {
+  cy.execInContainer({
+    command: 'systemctl reload httpd',
+    name: Cypress.env('dockerName')
+  });
+});
+
+Then('I can authenticate to the centreon platform', () => {
+  cy.loginByTypeOfUser({ jsonName: 'admin' });
+
+  cy.url().should('contain', '/monitor');
+});
+
+Then(
+  'the resource icons are displayed in configuration and monitoring pages',
+  () => {
+    cy.getByLabel({ label: 'State filter' }).click();
+
+    cy.get('[data-value="all"]').click();
+
+    cy.contains(service).parent().get('.MuiChip-root').should('contain', 's');
+
+    cy.contains(host).parent().get('.MuiChip-root').should('contain', 'h');
+
+    cy.navigateTo({
+      page: 'Hosts',
+      rootItemNumber: 3,
+      subMenu: 'Hosts'
+    });
+
+    cy.wait('@getTimeZone').then(() => {
+      cy.getIframeBody().contains(host).find('.ico-18').should('be.visible');
+    });
+
+    cy.navigateTo({
+      page: 'Services by host',
+      rootItemNumber: 3,
+      subMenu: 'Services'
+    });
+
+    cy.wait('@getTimeZone').then(() => {
+      cy.getIframeBody()
+        .contains(service)
+        .parent()
+        .find('.ico-14')
```

Comment:
you are checking the container of the image
then, if the image is broken, the test will still succeed
maybe you should check `src` attribute of the `img`

Reply:
done !

Reply:
Not done !
You can be inspired by https://stackoverflow.com/a/73604196
Please resolve the conversation only when it is really done